### ISSUE TITLE
Resolve XXXs

### ISSUE THREAD

**Issue by vmbrasseur** [vmbrasseur/10stepprogram#13]
There are `# XXX` lines in the outline. We should get those resolved.

**Comment by jberkus** [comment]
Most of those are due to indentation fail.  Fixing.

However: 

```
    The 6 basic presentation stories # XXX [citations needed] 
```

Um, what?

**Comment by vmbrasseur** [comment]
Where did you get the basic presentation story types?

**Comment by jberkus** [comment]
Ian and I started with five.  Based on one of my later presentations, I added one.  AFAIK, nobody else has codified them this way.  The concept is based on the seven story plots: https://en.wikipedia.org/wiki/The_Seven_Basic_Plots

**Comment by jberkus** [comment]
Brief summary:
- Catalog: listing of things "10 CSS tags you didn't know about"
- End-to-End: covering one thing from beginning to end, top to bottom, whatever.
- Solution Quest: first I tried this, then I tried that, then I succeeded.
- Enlightenment: I used to do this dumb thing.  Then I learned X and my life changed.
- Show & Tell: I'm going to create a replica in less than 5 minutes.  Then I'll show you how I did it.
- Theme & Variations: six ways to do REST services using Flask.
